src/test/java/com/google/cloud/tools/maven/run/AbstractRunMojoTest.java

```diff
@@ -0,0 +1,3 @@
+package com.google.cloud.tools.maven.run;
+
+public class AbstractRunMojoTest {}
```

Comment:
Remove it.

Reply:
oops, forgot to fill out this test class